src/pheval/post_processing/post_processing.py

```diff
@@ -449,6 +449,9 @@ def generate_pheval_result(
     Raises:
         ValueError: If the results are not all the same type or an error occurs during file writing.
     """
+    if not pheval_result:
+        print(f"Warning: No results found for {tool_result_path.name}")
```

Comment:
I dont know how far this goes but you should avoid at all time to use print() in favour of logging.warning or logging.info() so that the client can set their preferred reporting levels.